tests/e2e/util.go

```diff
@@ -4915,3 +4915,97 @@ func waitForVolumeSnapshotContentToBeDeleted(client snapclient.Clientset, ctx co
 	})
 	return waitErr
 }
+
+/*
+This method will fetch the master node IP where controller is running
+*/
+func getK8sMasterNodeIPWhereControllerLeaderIsRunning(ctx context.Context,
```

Comment:
`getK8sMasterNodeIPWhereControllerLeaderIsRunning()` should be `getK8sMasterNodeIPWhereContainerLeaderIsRunning()` and `controller_name` should be `container` right?